CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ project(pfs)
 
 option(BUILD_SHARED_LIBS "Build shared library (.so) instead of static one (.a)" ON)
 option(ADDRESS_SANITIZER "Enable address sanitizer" OFF)
+option(BUILD_SAMPLES "Build samples" ON)
+option(BUILD_TESTING "Build testing" ON)
```

Comment:
`option(BUILD_TESTS "Build tests" ON)`

Reply:
Hi @kfiros, I chose BUILD_TESTING, because that seems to be the standard cmake variable for that:
https://cmake.org/cmake/help/latest/module/CTest.html

However, if you prefer, I can adjust it to BUILD_TESTS. Please let me know!